com.unity.ml-agents/Runtime/Sensor/RayPerceptionSensorComponentBase.cs

```diff
@@ -37,14 +37,11 @@ public abstract class RayPerceptionSensorComponentBase : SensorComponent
         [Header("Debug Gizmos", order = 999)]
         public Color rayHitColor = Color.red;
         public Color rayMissColor = Color.white;
-        [Tooltip("Whether to draw the raycasts in the world space of when they happened, or using the Agent's current transform'")]
-        public bool useWorldPositions = true;
```

Comment:
I removed this; it was a bit confusing, and I think it won't matter at all after later drawing improvements (outside of this PR).